db/monitor/DELA0F3.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<!-- DELL UltraSharp 38 Monitor (Model 2017) -->
+<monitor name="DELL U8318DW" init="standard">
```

Comment:
Probably, just mistake: `DELL U8318DW`

According to [link](https://www.driverscloud.com/en/drivers/manufacturercat/90-4-28/dell-monitors), it's Dell U3818DW(DisplayPort), which matches the PR's name.

Reply:
yes, this is a mistake, I will fix it - sharp eyes ;-)